designsafe/static/scripts/data-depot/components/projects/pipeline-team/pipeline-team.template.html

```diff
@@ -104,8 +109,9 @@ <h5 style="font-family: Helvetica Neue">Citation Preview</h5>
                             <i class="fa fa-arrow-circle-down fa-2x" ng-click="$ctrl.orderMembers(false)"></i>
                         </td>
                         <td>
-                            <button class="btn btn-small btn-add" ng-if="!$ctrl.saved" ng-click="$ctrl.saveTeam()">Save</button>
-                            <button class="btn btn-small btn-add" ng-if="$ctrl.saved" disabled>Authors Saved</button>
+                            <button class="btn btn-small btn-add" ng-click="$ctrl.saveTeam()">Save</button>
+                            <!-- <button class="btn btn-small btn-add" ng-if="!$ctrl.saved" ng-click="$ctrl.saveTeam()">Save</button>
```

Comment:
Is there a reason you didn't just remove these commented out lines?

Reply:
Oops! I'll fix that.